src/dynarec/la64/dynarec_la64_00.c

```diff
@@ -193,6 +209,18 @@ uintptr_t dynarec64_00(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 SMWRITELOCK(lock);
             }
             break;
+        case 0x8B:
+            INST_NAME("MOV Gd, Ed");
+            nextop=F8;
+            GETGD;
+            if(MODREG) {
+                MVxw(gd, xRAX+(nextop&7)+(rex.b<<3));
```

Comment:
Probably need `TO_LA64`macro here

Reply:
Ah yes, sorry

Reply:
It's a bit annoying the register number can continuous.

Reply:
An alternative way could be also: `(nextop&7)+((rex.b<<3)?xR8:xRAX)` not sure it's usefull, but that can be a possibility.

Reply:
> It's a bit annoying the register number cannot be continuous.

I agree, but there is no choice :(